Source/engine/demomode.cpp

```diff
@@ -1,301 +1,295 @@
-/**
- * @file animationinfo.h
- *
- * Contains most of the the demomode specific logic
- */
-
```

Comment:
what happened to the header?

Reply:
Third commit -- this header did not belong here as it's a cpp file (the name was wrong anyway)